src/MSGtester.java

```diff
@@ -1,4 +1,5 @@
 import java.util.ArrayList;
+import java.util.Arrays;
```

Comment:
Same here, we all share this file if our code relies on changes here but we all need different changes then our code will break